extensions/azure-extensions/src/main/java/io/druid/storage/azure/CompressionUtilsWrapper.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Druid - a distributed column store.
+ *  Copyright 2012 - 2015 Metamarkets Group Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.druid.storage.azure;
+
+import com.google.common.base.Predicate;
+import com.google.common.io.ByteSource;
+import com.metamx.common.CompressionUtils;
+import com.metamx.common.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+public class CompressionUtilsWrapper
+{
+
+  public FileUtils.FileCopyResult unzip(
```

Comment:
My preferred method is to mock the connector as per https://github.com/metamx/druid/blob/queryTimeLookup/extensions/s3-extensions/src/test/java/io/druid/storage/s3/S3DataSegmentPullerTest.java 

If possible, to simply mocking the calls to AzureStorageContainer should yield a more "pure" unit test.